vocabularies/survey-status.ttl

```diff
@@ -0,0 +1,65 @@
+﻿@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix sstatus: <http://linked.data.gov.au/def/survey-status/> .
+
+<http://linked.data.gov.au/def/survey-status> a owl:Ontology .
+
+sstatus:conceptScheme a skos:ConceptScheme;
+  dct:created "2019-04-09T03:31:01.023Z"^^xsd:dateTime;
+  dct:modified "2019-04-09T03:34:27.949Z"^^xsd:dateTime;
+  dct:creator "Geological Survey of Queensland" ;
+  skos:definition "The stage in the lifecycle of conducting a survey."@en;
+  skos:prefLabel "Survey Status"@en .
+
+sstatus:intended a skos:Concept;
+  skos:definition "A survey has been proposed, but has not yet commenced."@en;
+  skos:notation "PRO"@en;
+  skos:inScheme sstatus:conceptScheme;
+  skos:prefLabel "Intended"@en;
+  skos:altLabel "Proposed"@en;
+  skos:topConceptOf sstatus:conceptScheme .
+
+sstatus:current-in-progress a skos:Concept;
+  skos:notation "INPRO"@en;
+  skos:definition "A survey is currently in progress."@en;
+  skos:inScheme sstatus:conceptScheme;
+  skos:prefLabel "In-progress"@en;
+  skos:topConceptOf sstatus:conceptScheme .
+
+sstatus:current-on-hold a skos:Concept;
+  skos:notation "ONHOL"@en;
+  skos:definition "A survey is current, but on-hold."@en;
+  skos:inScheme sstatus:conceptScheme;
+  skos:prefLabel "On-hold"@en;
+  skos:topConceptOf sstatus:conceptScheme .
+
+sstatus:interim-report-received a skos:Concept;
+  skos:notation "INRPT"@en;
+  skos:definition "An interim report for the survey has been received."@en;
+  skos:inScheme sstatus:conceptScheme;
+  skos:prefLabel "Interim report received"@en;
+  skos:topConceptOf sstatus:conceptScheme .
+
+sstatus:final-report-received a skos:Concept;
+  skos:notation "FINAL"@en;
+  skos:definition "The final report for the survey has been received."@en;
+  skos:inScheme sstatus:conceptScheme;
+  skos:prefLabel "Final report received"@en;
+  skos:topConceptOf sstatus:conceptScheme .
+
+sstatus:completed a skos:Concept;
+  skos:notation "COMP"@en;
+  skos:definition "The survey has been completed, and a final report has been received."@en;
```

Comment:
This should read, "...and a final report has **not** been received."
To differentiate it from concept final-report-received.